<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomCommandWidget</class>
 <widget class="QDockWidget" name="CustomCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Command</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="hLayoutInput">
      <item>
       <widget class="QLineEdit" name="commandLineEdit">
        <property name="placeholderText">
         <string>Enter command</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wrapCheckBox">
        <property name="text">
         <string>Wrap Lines</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="scrollTopCheckBox">
        <property name="text">
         <string>Scroll to Top</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     <item>
      <widget class="QComboBox" name="intervalComboBox">
       <property name="toolTip">
        <string>Auto-run interval (seconds); 0 disables</string>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
      </widget>
     </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="outputTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
